README.md

```diff
@@ -14,23 +14,49 @@ $ python setup.py install
 ```
 3- Verify if the driver is installed
 ```
-$ python verify_installation.py
+$ pip install entry_point_inspector
+$ epi group show oslo.messaging.notify.drivers
+
 ```
 Output in case of a successful instalation:
-`prometheus_exporter driver found.`
+`prometheus_exporter` is listed in the `Name` column and the `Error` column should be empty.
 Output in case of an unsuccessful instalation:
-`prometheus_exporter driver not found.`
-`Available drivers: ['log', 'messagingv2', 'noop', 'routing', 'test', 'messaging']`
+`prometheus_exporter` is listed in the `Name` column and the `Error` column will have more information.
 
 
 ### Configuration ###
 
-After install the driver you will need to update the :ironic.conf: and add
-:file_path: option (the file extension should be .json)
+After install the driver you will need to update the :ironic.conf: and add the below information.
 
 ```
+[conductor]
+send_sensor_data=true
+
 [oslo_messaging_notifications]
 driver = prometheus_exporter
 transport_url = fake://
-file_path=/tmp/ironic_prometheus_exporter/metrics.json
+location=/tmp/ironic_prometheus_exporter
+```
+
+
+### Running exporter application ###
+
+The Flask Application is responsible to merge all the metrics files present in the directory
+set in :[oslo_messaging_notifications]/location:.
+
+To run the Flask Application follow the steps listed below:
+1 - open the repository directory
+```
+$ cd ironic-prometheus-exporter/
+```
+2- set the FLASK_* environment variables and the location of the :ironic.conf: file.
+```
+$ export IRONIC_CONFIG=/etc/ironic/ironic.conf
+$ export FLASK_APP=ironic_prometheus_exporter/app/exporter.py
+$ export FLASK_RUN_HOST=$HOST_IP
+$ export FLASK_RUN_PORT=5000
+```
+3- run the Flask Application
+```
+$ python -m flask run &
```

Comment:
Please do not recommend this for production, people should use a real WSGI server.

Reply:
Ack I will see how to run in WSGI and update here

Reply:
I've added a NOTE "**NOTE:** if you want to deploy in production please check the Flask [documentation](http://flask.pocoo.org/docs/dev/deploying/)"